CPS-???/README.md

```diff
@@ -0,0 +1,42 @@
+---
+CPS: ?
+Title: On chain dApp / scripts audits
+Status: Open
+Category: dApps
```

Comment:
```suggestion
Category: Metadata
```
since there's no `dApps` category on the initial list, I'd suggest `Metadata` because it should refer to script certification as well.